### DISCUSSION TITLE
Trouble Building CBC Package

### DISCUSSION THREAD

**Discussion by eranayoner** [coin-or/coinbrew#92] [Q&A]
Hello, I receive an error when I try the commands on my terminal:
`wget -nH https://raw.githubusercontent.com/coin-or/coinbrew/master/coinbrew`
`chmod u+x coinbrew `
`bash coinbrew fetch Cbc@master --no-prompt`
`bash coinbrew build Cbc@stable/2.10`

Specifically a build error with code 77. The part of the message reads:

configure: error: in `/Users/elifrana/CBC/build/Data/Sample/master':
configure: error: C compiler cannot create executables

[config.log](https://github.com/coin-or/coinbrew/files/13545789/config.log)

I am also attaching the configure log. I suspect the error has to do with an architectural compatibility issue, as I am using an M1 mac with MacOS Version 14.1.2. I must also note that I am using a conda environment different from base to perform these operations. Yet I do not know how to fix it. Any help would be appreciated. Thanks in advance!

**Comment by svigerske** [comment]
configure finds that you have gcc, but there seems to be a problem when calling gcc to compile a simple C program:
```
$ gcc -O2 -DNDEBUG    conftest.c  >&5
-macosx_version_min has been renamed to -macos_version_min
ld: warning: ignoring duplicate libraries: '-lSystem'
ld: library 'System' not found
collect2: error: ld returned 1 exit status
```
You may want to check whether gcc is indeed working.

If you want different C/C++ compilers, then set variables `CC` and `CXX`.

**Reply by eranayoner** [reply]
Thank you so much, I made sure gcc was working. I also tried to compile with Clang to no avail. I apologize as I am not too familiar with building in C.  However I tried both methods below which terminated with Error Code 258.
```
CC=/usr/local/bin/gcc CXX=/usr/local/bin/g++ bash coinbrew fetch Cbc@master --no-prompt
bash coinbrew build Cbc@stable/2.10
``` 
`
CC=/usr/bin/clang CXX=/usr/bin/clang++ bash coinbrew fetch Cbc@master --no-prompt 
`
I am attaching the log once again. I did not have any issues downloading this package on my other Mac with an intel chip or any other computers with Windows as their OS.  
[config.log](https://github.com/coin-or/coinbrew/files/13547225/config.log)
